src/core/manager.c

```diff
@@ -4083,12 +4083,14 @@ void manager_set_show_status(Manager *m, ShowStatus mode) {
         if (!MANAGER_IS_SYSTEM(m))
                 return;
 
+        bool enabled = IN_SET(m->show_status, SHOW_STATUS_TEMPORARY, SHOW_STATUS_YES);
```

Comment:
I think this should be `mode` instead of `m->show_status`.

Reply:
Yep.